src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowFlag.java

```diff
@@ -8,6 +8,7 @@ public enum FlowFlag {
     PIP_ENABLED,
     NOTICE_OF_CHANGE,
     AGREED_TO_MEDIATION,
+    MULTI_CLAIM_YES,
```

Comment:
IS_MULTI_TRACK ?

Reply:
@HarryH96 are you suggesting that I should rename this field ?

Reply:
variable renamed and unit test added.